src/github/artifacts/artifacts.ts

```diff
@@ -0,0 +1,57 @@
+import { tmpdir } from 'os';
+import { githubConfig, ticsConfig } from '../../configuration';
+import { create } from '@actions/artifact';
+import { logger } from '../../helper/logger';
+import { readdirSync } from 'fs';
+import { join } from 'canonical-path';
+
+export async function uploadArtifact(): Promise<void> {
+  const artifactClient = create();
+
+  try {
+    logger.header('Uploading artifact');
+    const tmpDir = getTmpDir() + '/ticstmpdir';
```

Comment:
Why adding the additional suffix? getTmpDir should already take care of this I think. Then you never need to care about the suffix.

Reply:
I recall this was to prevent the additional directory in the zip. If we ever change this name, this plugin also needs to be adapted.